lightning/src/ln/payment_tests.rs

```diff
@@ -621,7 +621,7 @@ fn do_retry_with_no_persist(confirm_before_reload: bool) {
 	nodes[1].node.handle_update_fulfill_htlc(&nodes[2].node.get_our_node_id(), &htlc_fulfill_updates.update_fulfill_htlcs[0]);
 	check_added_monitors!(nodes[1], 1);
 	commitment_signed_dance!(nodes[1], nodes[2], htlc_fulfill_updates.commitment_signed, false);
-	expect_payment_forwarded!(nodes[1], nodes[0], nodes[2], None, false, false);
+	expect_payment_forwarded!(nodes[1], nodes[0], nodes[2], None, true, false);
```

Comment:
Isn't clear from the commit message why this is flipped. Was this what was incorrect?